<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WiVRn dashboard</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/wivrn.svg</normaloff>:/images/wivrn.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="banner_capsysnice">
      <property name="styleSheet">
       <string notr="true">.QFrame
{
background-color: rgb(255, 170, 127);
border: 1px solid rgb(255, 127, 127);
border-radius: 5px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="banner_capsysnice_icon">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="banner_capsysnice_text">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The server does not have CAP_SYS_NICE capabilities. Click &lt;a href=&quot;setcap&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;here&lt;/span&gt;&lt;/a&gt; to grant it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="banner_capsysnice_dismiss">
         <property name="icon">
          <iconset theme="window-close"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="banner_nvidia_layer">
      <property name="styleSheet">
       <string notr="true">.QFrame
{
background-color: rgb(255, 170, 127);
border: 1px solid rgb(255, 127, 127);
border-radius: 5px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="banner_nvidia_layer_icon">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="banner_nvidia_layer_text">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The &lt;a href=&quot;https://github.com/WiVRn/WiVRn/issues/180&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Monado NVIDIA layer&lt;/span&gt;&lt;/a&gt; is not installed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="banner_nvidia_layer_dismiss">
         <property name="icon">
          <iconset theme="window-close"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group_server">
      <property name="title">
       <string>Server status</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QStackedWidget" name="stacked_widget_server">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="server_stopped">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>:/images/wivrn.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="font">
                  <font>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Stopped</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="button_start">
                 <property name="text">
                  <string>Start</string>
                 </property>
                 <property name="icon">
                  <iconset theme="media-playback-start"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="server_running">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>:/images/wivrn.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="font">
                  <font>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Running</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="button_settings">
                 <property name="text">
                  <string>Settings</string>
                 </property>
                 <property name="icon">
                  <iconset theme="preferences-system-symbolic"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="button_stop">
                 <property name="text">
                  <string>Stop</string>
                 </property>
                 <property name="icon">
                  <iconset theme="media-playback-stop-symbolic"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="layout_authentication"/>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>For Steam games, set the launch options to:</string>
               </property>
               <property name="textFormat">
                <enum>Qt::TextFormat::PlainText</enum>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextInteractionFlag::NoTextInteraction</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
               <item>
                <widget class="QPushButton" name="copy_steam_command">
                 <property name="toolTip">
                  <string>Copy to clipboard</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset theme="edit-copy-symbolic"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_steam_command">
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string notr="true">PRESSURE_VESSEL_FILESYSTEMS_RW=$XDG_RUNTIME_DIR/wivrn/comp_ipc %command%</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::TextInteractionFlag::NoTextInteraction</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group_client">
      <property name="title">
       <string>Headset status</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0">
         <item>
          <widget class="QLabel" name="label_client_status">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true">client_status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_usb">
           <property name="text">
            <string>Connect by USB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_wizard">
           <property name="text">
            <string>Wizard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_disconnect">
           <property name="text">
            <string>Disconnect</string>
           </property>
           <property name="icon">
            <iconset theme="network-disconnect-symbolic"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_details">
           <property name="text">
            <string>Details</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_authentication" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,0,0,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_pin">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string notr="true">label_pin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_pair_headset">
            <property name="text">
             <string>Pair headset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_cancel_pairing">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_manage_headsets">
            <property name="text">
             <string>Manage headsets</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_how_to_connect">
         <property name="text">
          <string notr="true">connect_to_this_server</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="headset_properties" native="true">
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="7" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Eye gaze tracking</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Eye resolution</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Speaker</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Field of view</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_refresh_rates">
            <property name="text">
             <string notr="true">refresh_rates</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_eye_size">
            <property name="text">
             <string notr="true">eye_size</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Microphone</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Available refresh rates</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Supported codecs</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Hand tracking</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Preferred refresh rate</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Face tracking</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_prefered_refresh_rate">
            <property name="text">
             <string notr="true">preferrred_refresh_rate</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_speaker">
            <property name="text">
             <string notr="true">speaker</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_mic">
            <property name="text">
             <string notr="true">mic</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_field_of_view">
            <property name="text">
             <string notr="true">field_of_view</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_hand_tracking">
            <property name="text">
             <string notr="true">hand_tracking</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_eye_gaze_tracking">
            <property name="text">
             <string notr="true">eye_gaze_tracking</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="label_face_tracking">
            <property name="text">
             <string notr="true">face_tracking</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="label_codecs">
            <property name="text">
             <string notr="true">codecs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>104</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_about">
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset theme="help-about-symbolic"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_exit">
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="icon">
         <iconset theme="application-exit-symbolic"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_systray_show">
   <property name="icon">
    <iconset theme="settings-configure-symbolic"/>
   </property>
   <property name="text">
    <string>&amp;Show</string>
   </property>
  </action>
  <action name="action_systray_hide">
   <property name="icon">
    <iconset theme="preferences-system-symbolic"/>
   </property>
   <property name="text">
    <string>&amp;Hide</string>
   </property>
  </action>
  <action name="action_systray_exit">
   <property name="icon">
    <iconset theme="application-exit-symbolic"/>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../wivrn-dashboard.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>banner_capsysnice_dismiss</sender>
   <signal>clicked()</signal>
   <receiver>banner_capsysnice</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>banner_nvidia_layer_dismiss</sender>
   <signal>clicked()</signal>
   <receiver>banner_nvidia_layer</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>869</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>852</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
